scripts/search/update-medium-index.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config();
```

Comment:
since here you are requiring the algolia client which already require `dotenv` then is no need to require it here again